jaeger_client/tracer.py

```diff
@@ -209,18 +220,18 @@ def start_span(self,
                     operation_name=operation_name,
                     tags=tags, start_time=start_time, references=valid_references)
 
-        self._emit_span_metrics(span=span, join=rpc_server)
+        self._emit_span_metrics(span=span, join=bool(rpc_server))
```

Comment:
perhaps move to var declaration instead?

Reply:
Can you please give me an example what you mean?

Reply:
L182, `rpc_server: bool = ...`

Reply:
Need to use cast, but else it works